src/core/internal/hash.d

```diff
@@ -351,56 +407,52 @@ size_t hashOf(T)(scope const T val, size_t seed = 0) if (!is(T == enum) && __tra
     }
     else
     {
-        static if (T.sizeof <= size_t.sizeof && __traits(isIntegral, T))
-        {
-            static if (size_t.sizeof < ulong.sizeof)
-            {
-                //MurmurHash3 32-bit single round
-                enum uint c1 = 0xcc9e2d51;
-                enum uint c2 = 0x1b873593;
-                enum uint c3 = 0xe6546b64;
-                enum uint r1 = 15;
-                enum uint r2 = 13;
-            }
-            else
-            {
-                //Half of MurmurHash3 64-bit single round
-                //(omits second interleaved update)
-                enum ulong c1 = 0x87c37b91114253d5;
-                enum ulong c2 = 0x4cf5ad432745937f;
-                enum ulong c3 = 0x52dce729;
-                enum uint r1 = 31;
-                enum uint r2 = 27;
-            }
-            auto h = c1 * cast(UnqualUnsigned!T) val;
-            h = (h << r1) | (h >>> (typeof(h).sizeof * 8 - r1));
-            h = (h * c2) ^ seed;
-            h = (h << r2) | (h >>> (typeof(h).sizeof * 8 - r2));
-            return h * 5 + c3;
-        }
-        else static if (T.sizeof > size_t.sizeof && __traits(isIntegral, T))
-        {
-            static foreach (i; 0 .. T.sizeof / size_t.sizeof)
-                seed = hashOf(cast(size_t) (val >>> (size_t.sizeof * 8 * i)), seed);
-            return seed;
-        }
-        else
-        {
-            return bytesHashAlignedBy!T(toUbyte(val), seed);
```

Comment:
Where did this go?

Reply:
This branch was only reached when `__traits(isArithmetic, T) && !__traits(isFloating, T) && !__traits(isIntegral, T)` which doesn't describe any `T`.